### PR TITLE
feature/hbd-6683

### PR DESCRIPTION
**Pull request by BakJakub**

### CODE REVIEW

File: Handbid_iPad/MainPointApp/MainAppCoordinator.swift [hunk 1]
```diff
@@ -11,6 +11,7 @@ struct MainAppCoordinator: App {
 	@StateObject private var registrationCoordinator: Coordinator<RegistrationPage, Any?>
 	@StateObject private var mainContainerCoordinator: Coordinator<MainContainerPage, Any?>
 	private let dataManager: DataManager
+	@State private var selectedView: MainContainerTypeView = .auction
```
Comment (mprzypasniak99) [inline Handbid_iPad/MainPointApp/MainAppCoordinator.swift:14]: This variable is not used anywhere

File: Handbid_iPad/MainContainer/MainContainer.swift [hunk 2]
```diff
@@ -5,10 +5,11 @@ import SwiftUI
 
 struct MainContainer<T: PageProtocol>: View {
 	@EnvironmentObject private var coordinator: Coordinator<T, Any?>
+	@EnvironmentObject private var authManager: AuthManager
+	@StateObject var deviceContext = DeviceContext()
 	@State var isSidebarVisible: Bool = DeviceConfigurator.isSidebarAlwaysVisible
 	@State var selectedView: MainContainerTypeView
-	@StateObject var deviceContext = DeviceContext()
-	@EnvironmentObject private var authManager: AuthManager
+	@State var cancellables = Set<AnyCancellable>()
```
[the remark this answers is not in the document]
Comment (BakJakub) [reply Handbid_iPad/MainContainer/MainContainer.swift:12]: changed

File: Handbid_iPad/MainPointApp/MainAppCoordinator.swift (hunk 1 above)
Comment (BakJakub) [reply Handbid_iPad/MainPointApp/MainAppCoordinator.swift:14]: We will need this in a later part of the application

File: Handbid_iPad/MainContainer/MainContainer.swift (hunk 2 above)
[the remark this answers is not in the document]
Comment (mprzypasniak99) [reply Handbid_iPad/MainContainer/MainContainer.swift:12]: What I meant is that `@State` might cause view redrawing on change of data. I don't see how changes in set of cancellables should impact the drawing of the view

File: Handbid_iPad/MainPointApp/MainAppCoordinator.swift (hunk 1 above)
Comment (mprzypasniak99) [reply Handbid_iPad/MainPointApp/MainAppCoordinator.swift:14]: You can add it when it'll actually be needed. Now it's more prone to being forgotten or misunderstood during future work on the project

**Pull request merged**